pkg/apis/core/validation/validation.go

```diff
@@ -1232,6 +1232,10 @@ func validateFlexVolumeSource(fv *core.FlexVolumeSource, fldPath *field.Path) fi
 	allErrs := field.ErrorList{}
 	if len(fv.Driver) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("driver"), ""))
+	} else {
+		for _, msg := range validation.IsQualifiedName(fv.Driver) {
```

Comment:
if there's an existing pod with bad data like this, tightening this validation means the pod can become impossible to delete

Reply:
@liggitt Are we doing validation during the delete path? The spec files might change and that would be a very strict requirement right?

Reply:
Updates are sometimes required during the delete path. Not sure what you mean by spec files changing, but in general, we can never tighten validation in a way that causes previously valid API objects to be invalid.